src/emm/emm_cli.py

```diff
@@ -225,16 +225,23 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
 @cli.command(context_settings=dict(max_content_width=100))
 @click.option("--title", help="Title for the pull request.")
 @click.option("--body", help="Body for the pull request")
+@click.option(
+    "--remote",
+    default="origin",
+    help="Remote to push against for the pull request. The default would be 'origin'",
+)
 @click.pass_context
-def pull_request(ctx: click.Context, title: Optional[str], body: Optional[str]) -> None:
+def pull_request(
+    ctx: click.Context, title: Optional[str], body: Optional[str], remote: str
+) -> None:
     """
     Create a Github pull request for changes in the base repository and any enabled modules.
 
     NOTE: Before using this command, you have to authenticate to github by 'gh auth login'.
 
     A pull request will be created for the base repository and any enabled modules that contain
     changes. Additionally, a comment will be added to each pull request with links to the other
-    pull requests.
+    pull requests. To specify a remote to push against, `--remote` option can be used.
```

Comment:
The added sentence here seems randomly added on and doesn't seem to fit with the rest of the paragraph. We might want to make a new paragraph for it. 

Also, we should explicitly state that the code will be pushed to "origin" and that a different remote can be specified with the `--remote` option.